lms/djangoapps/experiments/flags.py

```diff
@@ -81,6 +81,13 @@ def __init__(
         ]
         self.use_course_aware_bucketing = use_course_aware_bucketing
 
+    @property
+    def _namespace(self):
+        """
+        Legacy property used to identify the app that uses this flag.
```

Comment:
This isn't a great docstring for the future. I'm thinking we should have `def _app_label` and `def _experiment_name` (or something like this), but that's just my opinion. 

For consideration: The docstring title should probably mention that will return metadata to be attached to the experiment event data. The docstring could also include a note around implementation with an example waffle flag, like `grades.my.experiment.waffle.flag`, where `app_label` will be `grades` and `experiment` will be `my.experiment.waffle.flag`.

In all cases, from the point of view of this class, this is not a "Legacy property".

Reply:
I agree. Please take a look at the new `_app_label` and `_experiment_name` properties.